### ISSUE TITLE
transportation-data-utils: Travis/CircleCI pipeline

### ISSUE THREAD

**Issue by sergiogcx** [cityofaustin/atd-data-tech#234]
As part of Epic #191 This issue is made to track the development of a pipeline in Travis

- [x] Retains monorepo architecture
- [x] Identifies changes to specific libraries
- [x] Builds libraries automatically
- [x] Shuts down build if package version already exists.
- [x] Can detect current environment: master (production), any other branch: dev
- [x] Test deployment to pypi under dev

**Comment by sergiogcx** [comment]
- I've set up a repo and I have connected it to CircleCI, it looks like it runs and it is properly configured to listen to any events to the two branches: master and dev. 

- The script can build packages, but nothing will be deployed until we have a well-tested environment detection script (currently working on this).

Also: Using CircleCI instead of Travis because lately travis has been very unreliable, often does not trigger builds at all and there have been outages. Using CircleCI (also free for open source) in the mean time.
